examples/iscsi-pd/iscsi-pd.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "iscsipd",
+    "kind": "Pod",
+    "apiVersion": "v1beta1",
```

Comment:
I see this comment is not addressed - we have started converting examples over to v1beta3...